test/kata7.getEmployerRole.test.js

```diff
@@ -1,5 +1,35 @@
-const { getEmployerRole } = require("../src");
+const { getEmployerRole } = require('../src');
 
-describe("getEmployerRole", () => {
-  test("returns the employee's role in the company", () => {});
+const employees = [];
+describe('getEmployerRole', () => {
+	test('check if anyone is employed', () => {
+		expect(getEmployerRole('Stuart', employees)).toBe('No one works here!');
+	});
+	test('check founder', () => {
+		employees.push({ name: 'Stuart', role: 'Founder' });
+		expect(getEmployerRole('Stuart', employees)).toBe('Founder');
+	});
+	test('check someone that does not work here', () => {
+		expect(getEmployerRole('James', employees)).toBe(
+			'James does not work here!'
+		);
+	});
+	test('check randoms', () => {
+		employees.push(
+			{ name: 'Satti', role: 'Developer' },
+			{ name: 'Jenny', role: 'Sales Associate' },
+			{ name: 'Javid', role: 'Human Recommended Reading Assistant' }
+		);
+		expect(getEmployerRole('Satti', employees)).toBe('Developer');
+		expect(getEmployerRole('Jenny', employees)).toBe('Sales Associate');
+		expect(getEmployerRole('Javid', employees)).toBe(
+			'Human Recommended Reading Assistant'
+		);
+	});
+	test('check duplicates', () => {
+		employees.push({ name: 'Satti', role: 'Admin' });
+		expect(getEmployerRole('Satti', employees)).toBe('Developer Admin');
+		employees.push({ name: 'Satti', role: 'Cleaner' });
+		expect(getEmployerRole('Satti', employees)).toBe('Developer Admin Cleaner');
+	});
 });
```

Comment:
Having one `employees` array that is used by each test that you keep adding to (`.push`) is bad practice. It could lead to unexpected problems down the line. For now, it would be better to define the `employees` array in each test with it's own variables. However, this will mean you'll have a lot of code repetition. Further into the track we will go over some ways you can reduce this repetition to make your code more DRY.